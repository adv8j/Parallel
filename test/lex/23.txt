parallel(threads=4) {
    process_data(data);
}
