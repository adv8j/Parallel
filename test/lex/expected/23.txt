IDENTIFIER
LPAREN
IDENTIFIER
ASSIGN
NUMBER
RPAREN
LBRACE
IDENTIFIER
LPAREN
IDENTIFIER
RPAREN
SEMICOLON
RBRACE
